src/components/organisms/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,21 @@
+import GalleryGrid from "../../molecules/GalleryGrid/GalleryGrid";
+import "../Gallery/Gallery.css";
+import { GalleryImagesData } from "../../molecules/GalleryGrid/GalleryGrid";
+
+interface GalleryProps {
+  title: string;
+  subtitle: string;
+  galleryImages: GalleryImagesData[];
+}
+
+const Gallery = (props: GalleryProps) => {
+  const Gallery = props;
```

Comment:
Props is not Class/constant/enum, please use camelCase, also avoid shadowing variables (changing to camelCase will fix this too)

Reply:
Changed